src/diart/console/serve.py

```diff
@@ -5,10 +5,8 @@
 
 from diart import argdoc
 from diart import models as m
-from diart import sources as src
 from diart import utils
-from diart.inference import StreamingInference
-from diart.sinks import RTTMWriter
+from diart.handler import StreamingHandlerConfig, StreamingHandler
```

Comment:
I don't like the term "handler" for these classes. The reason is that `StreamingHandler` does not necessarily make it clear that you're streaming through the network, or even that you're running a pipeline over a server.
What about `WebSocketStreamingServer`? (The config class should disappear as I mentioned before)

Reply:
renamed